blkn/src/Button/IconWrapper.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import * as React from "react";
+import { colors } from "../constants";
+
+type Props = {
+  size: "normal" | "small" | "large",
+  type: "primary" | "secondary",
+  icon: Function,
```

Comment:
nope, too vague. be specific

Reply:
The right type here would be something like `React.ComponentType<{ height: number, fill: string }>`. But I'm not so sure if we want to use this pattern (`<SomeComponent icon={SomeIcon} />`), it seems unusual.

[In the Header component](https://github.com/kiwicom/react-kiwi-ui/blob/master/blkn/src/Container/Header.js), we're passing a `React.Node` (like `<SomeComponent icon={<SomeIcon height={32} />} />`), but that's not applicable here, because the size and color are concerns of the button.

Maybe we can use a render prop for this icon (like `<SomeComponent icon={(size, color) => <SomeIcon height={size} fill={color}) />`), what do you think @oreqizer about this?

Reply:
I also stopped at this 'unusual' patter, I was considering render props and compound component pattern. But I like it because of it less verbose than others.